src/api/v1/endpoint/chat.py

```
@@ -1,17 +1,85 @@
+
+        # Pydantic 모델로 변환
+        compare_result = CompareResult(**result_data)
+
+        return CommonResponse(
+            isSuccess=True,
+            code="AI200",
+            message="비교 분석 성공입니다.",
+            result=compare_result,
+        )
+
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)